src/http_rest_api_handler.cpp

```diff
@@ -402,16 +402,19 @@ Status validateContentFieldsEmptiness(::inference::ModelInferRequest_InferInputT
     return StatusCode::OK;
 }
 
-Status addBinaryInputs(::inference::ModelInferRequest& grpc_request, const char* binary_inputs, size_t binary_inputs_size) {
+Status handleBinaryInputs(::inference::ModelInferRequest& grpc_request, const std::string request_body, size_t endOfJson) {
```

Comment:
Wouldn't it be more performant if we passed `request_body` via reference instead of value?